CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## v4.0.0 (2020-09-03)
+
+## Breaking changes
+* `sessionDomain` is required
+* Removed `siteSpecificLogout` param- there is no option to disable site specific logout
+* Drop `hasProduct` and `hasSubscription` functions- replaced by `hasAccess`
+
```

Comment:
Might be good to also mention the removal of `autologin` param for `hasSession()` (caused by the removal of spid/hassession as fallbacks).

Reply:
✅